src/Foundation/Features/Search/_FacetContent.cshtml

```diff
@@ -1,8 +1,12 @@
-﻿@using EPiServer.Find
+﻿@using EPiServer
+@using EPiServer.Find
+@using EPiServer.ServiceLocation
 @using EPiServer.Shell.Web.Mvc.Html
+@using Foundation.Cms.Categories
 @model Foundation.Demo.ViewModels.DemoSearchViewModel<SearchResultPage>
 @{
     Layout = null;
+    var _contentRepository = ServiceLocator.Current.GetInstance<IContentRepository>();
```

Comment:
I dont link having this can we have the viewmodel populate the id and the category?

Reply:
I think it's easiest way to render the category. If using view model to populate the category, you need create a new model like FacetGroupOption and FacetOption for the only Category case.

Reply:
What about storing the category names for the language instead of array of integer?

Reply:
Index of categories is also used for another feature (tag of Location Item Page, need storing the Id to get the link, etc... the name is not enough)